v2/pkg/commands/build/build.go

```diff
@@ -262,9 +262,12 @@ func execBuildApplication(builder Builder, options *Options) (string, error) {
 
 	// Compile the application
 	printBulletPoint("Compiling application: ")
+	outputFile := builder.OutputFilename(options)
+	options.ProjectData.OutputFilename = outputFile
+	options.ProjectData.Name = outputFile
+	options.ProjectData.Info.ProductName = outputFile
```

Comment:
Why are these two being set here? I'd want my structure to be `/Applications/{{.Name}}.app/Contents/MacOS/{{.Outputfilename}}`. This is because I want the app name to be capitalized and the executable name to be lower case. So `/Applications/MyApp.app/Contents/MacOS/myapp`.